### PR TITLE
feat: add variable `os_disk_size_gb`

### LINKED ISSUE #132: Add os_disk_size_gb variable to VM module
### Module Name

vm

### Description

Add variable os_disk_size_gb parameter to VM module to control the size of the OS disk.

### Example Terraform Configuration

```terraform
variable "os_disk_size_gb" {
  description = "The name of this OS disk."
  type        = number
  default     = null
}

....
os_disk {
    name                 = var.os_disk_name
    caching              = var.os_disk_caching
    disk_size_gb         = var.os_disk_size_gb
    storage_account_type = var.os_disk_storage_account_type
}
....
```

### PR DESCRIPTION
**Pull request by stiross79**
Fixes https://github.com/equinor/terraform-baseline/issues/132

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -99,6 +99,12 @@ variable "os_disk_storage_account_type" {
   default     = "StandardSSD_LRS"
 }
 
+variable "os_disk_size_gb" {
+  description = "The size of this OS disk."
```
Comment (hknutsen) [inline variables.tf]: We could clarify unit in description:

```suggestion
  description = "The size of this OS disk in gigabytes (GB)."
```

Comment (stiross79) [reply variables.tf]: Good catch

**Review by stiross79 — COMMENTED** [review]
Agreed.

**Review by hknutsen — APPROVED** [review]
Looks good 🎉

**Pull request merged**